From 487ade98b77483b41b9bc3ff415d26e64fc9137a Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Thu, 29 Mar 2018 14:22:04 +0800
Subject: [PATCH] arm64: dts: rockchip: exclude rk3308-evb-ext-v10 from rk3308
 evb boards

Exclude rk3308-evb-ext-v10.dtsi from rk3308 evb boards, that it's only
used for IP functional verification.

Change-Id: Ic679da54e16134e6326b50fd7de4fbafab57ac8f
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts     | 1 -
 arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts | 1 -
 arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts | 1 -
 3 files changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
index f121f3b0dd16..804e8d2790aa 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
@@ -7,7 +7,6 @@
 /dts-v1/;
 
 #include "rk3308-evb-v10.dtsi"
-#include "rk3308-evb-ext-v10.dtsi"
 
 / {
 	model = "Rockchip RK3308 evb analog mic board";
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts
index d1fd0ccb7a59..2ba196c99d88 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts
@@ -7,7 +7,6 @@
 /dts-v1/;
 
 #include "rk3308-evb-v10.dtsi"
-#include "rk3308-evb-ext-v10.dtsi"
 
 / {
 	model = "Rockchip RK3308 evb digital-i2s mic board";
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts
index 4a0f1a619d6f..68476c796b44 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts
@@ -7,7 +7,6 @@
 /dts-v1/;
 
 #include "rk3308-evb-v10.dtsi"
-#include "rk3308-evb-ext-v10.dtsi"
 
 / {
 	model = "Rockchip RK3308 evb digital-pdm mic board";
-- 
2.35.3

